libs/datasets/dataset_utils.py

```diff
@@ -142,6 +142,18 @@ def check_index_values_are_unique(data, index=None, duplicates_as_error=True):
     return None
 
 
+def decorate_new_cases(multiregion_timeseries):
+    """Adds a new_cases column to the dataset by calculating the daily diff in cases.
+
+    Args:
+        multiregion_timeseries: MultiRegionTimeseriesDataset
+
+    """
+    df = multiregion_timeseries.groupby_region()
```

Comment:
nit: `df` is a some kind of special groupby object, not a normal data frame.

Reply:
Understood.